fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/InstrumentedTransaction.java

```diff
@@ -191,16 +191,6 @@ public void setReadVersion(long l) {
         underlying.setReadVersion(l);
     }
 
-    @Override
-    public CompletableFuture<Long> getEstimatedRangeSizeBytes(final byte[] begin, final byte[] end) {
```

Comment:
This is technically an unrelated change, but there were duplicate identical implementations of this method in `InstrumentedTransaction` and `InstrumentedReadTransaction`, even though `InstrumentedTransaction extends InstrumentedReadTransaction`, so I consolidated to one place. Could remove or make it a separate change.